projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -81,13 +82,19 @@ def configure_initial_logging_before_anything():
     3. When initialize_job hook (applicable for vdk run only) is executed then below configure_loggers function
         is run which adds more context to the logs and initializes syslog handler (if configured to do so)
     """
-    log_level = "WARNING"
-    if os.environ.get(LOG_LEVEL_VDK, None):
-        log_level = os.environ.get(LOG_LEVEL_VDK)
-    elif os.environ.get("VDK_LOG_LEVEL_VDK", None):
-        log_level = os.environ.get("VDK_LOG_LEVEL_VDK")
-
-    logging.basicConfig(format="%(message)s", level=logging.getLevelName(log_level))
+    warnings.warn(
```

Comment:
Please don't make changes to core and plugins in the same PR. Plugin tests don't run against vdk-core that was built from source and this often leads to confusion. This particular change won't cause a problem, because there are no tests related to it.

Please move the core change to a separate PR. Maybe we don't even need to merge it right now, because structlog is not fully ready yet.

Reply:
ok, moving it to another PR.